.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -884,6 +884,17 @@ do
 done
 
 
+#############################################################################
+# Cleanup Old Email
+
+# Will scan the emails table in the main Submitty database for email
+# receipts that were successfully sent at least 360 days ago, with no
+# errors, and delete them from the table.  A maximum of 10,000 email
+# receipts will be deleted.
+if [ "${WORKER}" == 0 ]; then
+    "${SUBMITTY_INSTALL_DIR}/sbin/cleanup_old_email.py" 360 10000
```

Comment:
For my own curiosity, why not put this in a cron task that runs monthly (or bi-monthly or whatever) as opposed to running it only when the installer gets run? That would make things a bit more consistent on when emails get removed if someone was wanting to look at old email info.

Reply:
Yes, I considered that.  And the sysadmin is free to create a cron job that calls it as a standalone script.  

But I decided against adding one more daily (or weekly) burden to the system and database [when would we run this?  we already have lots of miscellaneous things happen at 'off' times].  We [ and I'd guess other universities would also ] perform 'cleanup' tasks or other destructive actions as per semester thing or 'when we update' thing.  We allowed emails to accumulate for 3(?) years before finally clearing them out.  I don't think there's an urgency to do it as a cron task.